introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone_dist_script_mode.ipynb

```diff
@@ -47,8 +47,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ensure sagemaker version >= 1.56.3\n",
-    "!pip show sagemaker"
+    "# ensure sagemaker version >= 1.65.1\n",
```

Comment:
Is this version upgrade needed i.e. do we only need 1.56.3 or higher?

Reply:
@iyerr3 Yes this is the version that is needed to be able to pull successfully the docker image v 1.0-1. We need SageMaker SDK V >= 1.65.1